samples/YoctoInstructions.md

```diff
@@ -0,0 +1,66 @@
+# .NET Core on Yocto distribution
+
+This document describes how to get .Net apps running on Yocto distribution. 
+At this momment the focus is on getting standalone applications running. 
+Unless explicitly mentioned, x86_64 and ARM platforms are supported. 
+
+## Getting started with base OS image
+Instructions bellow assume familiarity with Yocto build process. 
+Initial testing as been done on 2.2 Morty but it is probably applicable to 
+other versions as well. 
+
+add following lines your local.conf or custom recipe.
+
+* .NET dependencies
+```
+CORE_IMAGE_EXTRA_INSTALL += "libunwind icu libcurl openssl"
+```
+
+* It is strongly recommended to use curl with OpenSSL backend. 
+You can add something like:
+
+```
+PACKAGECONFIG_pn-curl = 'zlib ipv6 ssl'
+```
+
+Check curl recipe for comple set of options. For debugging you may also add
+
+```
+CORE_IMAGE_EXTRA_INSTALL += "curl" to get command line tool.
+```
+
+* Some extra space will be needed for the app. Adjust number bellow to fit the need. 
+This simply adds 1G of extra space. 
+
+```
+IMAGE_ROOTFS_EXTRA_SPACE_append += "+ 10000000"
+```
+
+On x86_64 .NET uses /lib64/ld-linux-x86-64.so.2. However Yocto defaults everything to /lib.
+This can be solved by adding symbolic link or adding following lines to force multi-lib layout similar to desktop Linux distributions.
+
+```
+require conf/multilib.conf
+MULTILIBS = "multilib:lib64"
+DEFAULTTUNE_virtclass-multilib-lib64 = "x86"
+```
+
+# Getting app ready
+
+Write and debug your app. When ready to publish, use:
+
+```
+dotnet publish -r <runtime identifier>
+```
+
+where indentifier is 'linux' or 'linux-arm' depending on your target architecture. 
+For ARM and more details you can take a look at [RaspberryPiInstructions](RaspberryPiInstructions.md).
+
+Package `bin/Debug/netcoreapp2.0/<runtime identifier>/publish` to your image.
+That directory has native executable binary as well as all needed runtime dependencies. 
+
+Getting standalone app has no steps specific to Yocto.
```

Comment:
The intention was to explicitly say that standalone app on Yocto works same way as standalone app on any other distribution. Maybe that is implicitly assumed but from now on, generic instructions can be followed. If we have any generic pointer that may be handy here.